CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## Version 0.18.0
+
+### Summary
+
+This release cycle focused on review and approval of two RFCs, one for Readout Error Mitigation (REM) [#1387](https://github.com/unitaryfund/mitiq/issues/1387) and one for Identity insertion noise scaling [#335](https://github.com/unitaryfund/mitiq/issues/335) (not listed as PRs). It also includes bug fixes and minor dependency upgrades. 
+
+### All Changes
+
+- Update black requirement from ~=22.6 to ~=22.8  infrastructureFor issues related to building, packaging, and continuous integration.
+#1477 by [@dependabot] 
+
+- docs: add Aaron-Robertson as a contributor for ideas, code, doc #1464 by [@allcontributors]
+
+- docs: add 1ucian0 as a contributor for infra, code #1461 by [@allcontributors]
+
+- Update amazon-braket-sdk requirement from ~=1.29.0 to ~=1.29.2  infrastructureFor issues related to building, packaging, and continuous integration. #1459 by [@dependabot] 
```

Comment:
Maybe we can manually remove all the "`infrastructureFor issues related to building, packaging, and continuous integration`" after each dependabot commit.  If we want to keep them, we need a space `infrastructureFor` ---> `infrastructure for `.